src/rosdiscover/recover/call.py

```diff
@@ -236,8 +240,11 @@ def is_unknown(self) -> bool:
         # NOTE same comment about default
         return self.param.is_unknown()
 
+    def __str__(self) -> str:
+        return f"ros::param::read(param={self.param}, default={self.default}"
```

Comment:
looks like you're missing a closing paren at the end of the string

Reply:
fixed